### PR TITLE
Швецов Артём

### PR DESCRIPTION
**Pull request by shvetsovart**
https://codesandbox.io/p/devbox/r6c8v9?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522cm0xuenae0007356kj2rtpzfp%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522cm0xuenad0002356kv0copc7z%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522cm0xuenad0004356kok4zx29g%2522%257D%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522cm0xuenad0006356kdr0npbep%2522%257D%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522cm0xuenad0002356kv0copc7z%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0xuenac0001356kxiemkl7u%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Findex.html%2522%252C%2522state%2522%253A%2522IDLE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A153%252C%2522startColumn%2522%253A8%252C%2522endLineNumber%2522%253A153%252C%2522endColumn%2522%253A8%257D%255D%257D%255D%252C%2522id%2522%253A%2522cm0xuenad0002356kv0copc7z%2522%252C%2522activeTabId%2522%253A%2522cm0xuenac0001356kxiemkl7u%2522%257D%252C%2522cm0xuenad0006356kdr0npbep%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0xuenad0005356k7z87zz6n%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522start%2522%252C%2522port%2522%253A5000%257D%255D%252C%2522id%2522%253A%2522cm0xuenad0006356kdr0npbep%2522%252C%2522activeTabId%2522%253A%2522cm0xuenad0005356k7z87zz6n%2522%257D%252C%2522cm0xuenad0004356kok4zx29g%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522cm0xuenad0003356ksqtd9sfl%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522start%2522%257D%255D%252C%2522id%2522%253A%2522cm0xuenad0004356kok4zx29g%2522%252C%2522activeTabId%2522%253A%2522cm0xuenad0003356ksqtd9sfl%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                <p>Yours sincerely,</p>
+                <img src="signature.png" alt="signature" width="150px">
+                <p>Artyom Shvetsov, <br>Future Student</p>
+                
+                <hr>
+            </footer>
+        </article>
+    
+        <footer>
+            <h5>HOGWARTS SCHOOL of HTML & CSS</h5>
```
Comment (mokhov) [inline index.html]: Мне кажется что футер не лучшее место для загловков

File: index.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                </p>
+            </section>
+
+            <footer> <!-- Farewell words -->
+                <p>
+                    Thank you for your attention, I'm really looking forward to continue to learn frontend development in your courses!
+                </p>
+                
+                <p>Yours sincerely,</p>
+                <img src="signature.png" alt="signature" width="150px">
```
Comment (mokhov) [inline index.html]: Можно не указывать "px"

File: index.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+                </p>
+
+                <p>
+                    But forget about all this motivational crap. Let me tell you some interesting facts about me & my life:
+                    <ul>
+                        <li>I'm 21</li>
+                        <li>Last year I ran a marathon</li>
+                        <li>The A in History I got for:
+                            <ol>
+                                <li>building a website <ruby><span style="color:grey">(on Tilda)</span><rt>[I'm sorry]</rt></ruby></li>
```
Comment (mokhov) [inline index.html]: 👍

File: index.html [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+            </footer>
+        </article>
+    
+        <footer>
+            <h5>HOGWARTS SCHOOL of HTML & CSS</h5>
+            <h6>Headmaster: Oleg Mokhov,<br>(Order of Berners-Lee, First Class, D.H.C.,<br>
+                J.S.(sorc.), International Confed. of Frontenders)</h6>
+            <hr>
+            <address>
+                <p><mark>Please contact me via email or Telegram for further communication</mark> <small>(my roommates get scared when something flies through our window):</small></p>
```
Comment (mokhov) [inline index.html]: Тег small не запрещен, но считается устаревшим

**Review by mokhov — COMMENTED** [review]
Есть что исправить

File: index.html [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+                </p>
+
+                <p>
+                    But forget about all this motivational crap. Let me tell you some interesting facts about me & my life:
+                    <ul>
+                        <li>I'm 21 years of <var>age</var></li>
+                        <li>Last year I ran a marathon</li>
+                        <li>The A in History I got for:
+                            <dl>
+                                <dt>- Building a website <ruby><span style="color:grey">(on Tilda)</span><rt>[I'm sorry]</rt></ruby></dt>
```
Comment (mokhov) [inline index.html]: Не самое лучшее применение dd-dt. Обычно их применяют в списке терминов.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+                </p>
+            </section>
+
+            <footer> <!-- Farewell words -->
+                <p>
+                    Thank you for your attention, I'm really looking forward to continue to learn frontend development in your courses!
+                </p>
+                
+                <p>Yours sincerely,</p>
+                <img src="signature.png" alt="signature" width="150">
```
Comment (mokhov) [inline index.html]: alt текст может быть понятнее, например «Подпись автора»

File: index.html [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+                <p>
+                    I am pleased to have been informed that I have been accepted at <q><strong>Hogwarts School of HTML and CSS</strong></q>. <br>
+                    Frankly speaking, my parents aren't frontenders. And neither am I... 
+                    In this letter I'd like to tell you the truth about me.
+                </p>
+            </section>
+
+            <section> <!-- Main part -->
+                <p>
+                    I am not the best <abbr>CT ITMO</abbr> (Computer Technologies faculty in ITMO [ITMO is not <br>
```
Comment (mokhov) [inline index.html]: У abbr стоит указывать атрибут title https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr

**Review by mokhov — COMMENTED** [review]
Ещё чуть-чуть

**Comment by mokhov** [comment]
Есть что ещё улучшить. Но ОК, принято

**Comment by shvetsovart** [comment]
Спасибо! А можете, пожалуйста, вкратце сказать, что ещё можно улучшить?

чт, 12 сент. 2024 г. в 18:14, Oleg Mokhov ***@***.***>:

**Pull request closed without merging**